low-level/files/opt/seagate/sspl/setup/sspl_test.py

```diff
@@ -167,12 +170,19 @@ def process(self):
             shutil.copyfile(sspl_test_backup, sspl_test_file_path)
             if rc != 0:
                 raise TestException("%s - ERROR: %s - CMD %s" % (self.name, error, CMD))
+
+            print('Starting the SSPL..')
+            CMD = "systemctl restart sspl-ll"
```

Comment:
why sspl restarting here

Reply:
we are resetting the config values in sspl.conf after the tests are completed so we need a restart here. I don't know why it was removed.
for code coverage also it is required.